ksql-common/src/test/java/io/confluent/ksql/util/timestamp/PartialStringToTimestampParserTest.java

```diff
@@ -43,44 +43,44 @@ public void init() {
   @Test
   public void shouldParseYear() {
     // When:
-    assertThat(parser.parse("2017"), is(fullParse("2017-01-01T00:00:00.000")));
+    assertThat(parser.parse("2017"), is(fullParse("2017-01-01T00:00:00.000+0000")));
```

Comment:
we shouldn't need all of these `+0000`s after this change, right?

Reply:
Yes we do. The 'fullParse' is expecting a tz - this makes the tests very explicit as to what timezone is expected.